src/util.ts

```diff
@@ -162,12 +163,12 @@ export function notifyCommandCompletedSuccessfully(): void {
   })
 }
 
-export async function askForChectlUpdateIfNeeded(): Promise<void> {
+export async function askForChectlUpdateIfNeeded(command: Command): Promise<void> {
   const ctx = ChectlContext.get()
   if (await VersionHelper.isChectlUpdateAvailable(ctx[ChectlContext.CACHE_DIR])) {
-    cli.info('A newer version of chectl is available.')
-    if (await cli.confirm('To deploy the latest version of Eclipse Che you have to update chectl first [y/n]')) {
-      cli.info('Please run "chectl update" and then repeat "server:deploy" command.')
+    cli.info('A newer version of chectl is available. To deploy the latest version of Eclipse Che you have to manually update chectl first.')
+    if (!await cli.confirm('Do you want to continue using older version now ? [y/n]')) {
```

Comment:
This seems backwards. What happens if they choose y/n here? What about flipping it.  "Do you want to update chectl now  [y/n]" and if they say y then give them the command to do so. If they say no then open the old version.